ddtrace/commands/ddtrace_run.py

```diff
@@ -10,6 +10,11 @@
 if debug and debug.lower() == "true":
     logging.basicConfig(level=logging.DEBUG)
 
+# Do not use `ddtrace.internal.logger.get_logger` here
```

Comment:
Are there any other places in our code one might want to disable rate limiting, in which case we might want to add that as an optional argument to `ddtrace.internal.logger.get_logger`?

Reply:
Not really, we want to use it everywhere we can, this is just one exception where we don't care because 1) we don't want to import `ddtrace` yet and 2) we know how the logger is used and it isn't going to be spammy